benchmark-ds.json

```diff
@@ -0,0 +1,108 @@
+{
+  "description": "aws 30 nodes on 3 shard in one region",
+  "client": {
+    "num_vm": 0,
+    "type": "t3.medium",
+    "regions": "iad"
+  },
+  "leader": {
+    "num_vm": 3,
+    "type": "t3.small",
+    "regions": "iad",
+    "protection": false,
+    "root": 15
+  },
+  "explorer_node": {
+    "num_vm": 3,
+    "type": "t3.small",
+    "regions": "iad",
+    "root": 25,
+    "protection": false
+  },
+  "bootnode": {
+     "enable": true,
+     "server": "52.40.84.2",
+     "name": "b6.harmony.one",
+     "port": 9876,
+     "user": "ec2-user",
+     "key": "oregon-key-benchmark.pem",
+     "p2pkey": "b2-9870.key",
+     "log_conn": false
+  },
+  "bootnode1": {
+     "enable": false,
+     "server": "54.86.126.90",
+     "name": "b5.harmony.one",
+     "port": 9867,
```

Comment:
usually, we would use the same port for both bootnodes. are you ok with different ports?

Reply:
that's okay, not a big issue though.